docs/en/integrations/data-ingestion/apache-spark/index.md

```diff
@@ -426,68 +436,335 @@ One of the most used data sources supported by Spark is JDBC.
+
+```sql
+   CREATE TEMPORARY VIEW jdbcTable
+           USING org.apache.spark.sql.jdbc
+           OPTIONS (
+                   url "jdbc:ch://localhost:8123/default", 
+                   dbtable "schema.tablename",
+                   user "username",
+                   password "password",
+                   driver "com.clickhouse.jdbc.ClickHouseDriver" 
+           );
+           
+   SELECT * FROM jdbcTable;
+```
+
+</TabItem>
+</Tabs>
 
 ### Write data
 
-:::important
-As of today, you can insert data using JDBC only into existing tables. 
-:::
+
+<Tabs groupId="spark_apis">
+<TabItem value="Java" label="Java" default>
 
 ```java
-    public static void main(String[] args) {
+ public static void main(String[] args) {
         // Initialize Spark session
         SparkSession spark = SparkSession.builder().appName("example").master("local").getOrCreate();
 
         // JDBC connection details
         String jdbcUrl = "jdbc:ch://localhost:8123/default";
         Properties jdbcProperties = new Properties();
         jdbcProperties.put("user", "default");
-        jdbcProperties.put("password", "******");
+        jdbcProperties.put("password", "123456");
+
         // Create a sample DataFrame
         StructType schema = new StructType(new StructField[]{
                 DataTypes.createStructField("id", DataTypes.IntegerType, false),
                 DataTypes.createStructField("name", DataTypes.StringType, false)
         });
-        
+
         List<Row> rows = new ArrayList<Row>();
         rows.add(RowFactory.create(1, "John"));
         rows.add(RowFactory.create(2, "Doe"));
 
+
         Dataset<Row> df = spark.createDataFrame(rows, schema);
 
+        //---------------------------------------------------------------------------------------------------
+        // Write the df to ClickHouse using the jdbc method
+        //---------------------------------------------------------------------------------------------------
+
         df.write()
                 .mode(SaveMode.Append)
-                .jdbc(jdbcUrl, "my_table", jdbcProperties);
-        // Show the DataFrame
-        df.show();
+                .jdbc(jdbcUrl, "example_table", jdbcProperties);
+
+        //---------------------------------------------------------------------------------------------------
+        // Write the df to ClickHouse using the save method
+        //---------------------------------------------------------------------------------------------------
+
+        df.write()
+                .format("jdbc")
+                .mode("append")
+                .option("url", jdbcUrl)
+                .option("dbtable", "example_table")
+                .option("user", "default")
+                .option("password", "123456")
+                .option("SaveMode", "append")
+                .save();
+
 
         // Stop the Spark session
         spark.stop();
     }
 ```
+</TabItem>
+<TabItem value="Scala" label="Scala">
+
+```java
```

Comment:
this should be scala

Reply:
Unfortunately, there is no `scala` option, so I used the closest one with the closest highlighting.